tools/docker-compose/README.md

```diff
@@ -27,7 +27,7 @@ Here are the main `make` targets:
 
 Notable files:
 
-- `tools/docker-compose/inventory` file - used to configure the local AWX development deploymen
+- `tools/docker-compose/inventory` file - used to configure the local AWX development.
```

Comment:
I would have rather fixed the typo. This change doesnt seem grammatically correct. Maybe this:

```suggestion
- `tools/docker-compose/inventory` file - used to configure the AWX development environment.
```